spec/controllers/socure_webhook_controller_spec.rb

```diff
@@ -54,202 +38,175 @@
         and_return(socure_secret_key_queue)
       allow(IdentityConfig.store).to receive(:socure_enabled).
         and_return(socure_enabled)
+      allow(SocureDocvResultsJob).to receive(:perform_later)
 
       stub_analytics
     end
 
     context 'webhook authentication' do
-      it 'returns OK and logs an event with a correct secret key and body' do
-        request.headers['Authorization'] = socure_secret_key
-        post :create, params: webhook_body
-
-        expect(response).to have_http_status(:ok)
-        expect(@analytics).to have_logged_event(
-          :idv_doc_auth_socure_webhook_received,
-          created_at: '2020-01-01T00:00:00Z',
-          customer_user_id: '123',
-          event_type: 'TEST_WEBHOOK',
-          reference_id: 'abc',
-          user_id: '123',
-        )
-      end
-
-      it 'returns OK with an older secret key' do
-        request.headers['Authorization'] = socure_secret_key_queue.last
-        post :create, params: webhook_body
-
-        expect(response).to have_http_status(:ok)
-      end
-
-      it 'returns unauthorized with a bad secret key' do
-        request.headers['Authorization'] = 'ABC123'
-        post :create, params: webhook_body
-
-        expect(response).to have_http_status(:unauthorized)
-      end
-
-      it 'returns unauthorized with no secret key' do
-        post :create, params: webhook_body
-
-        expect(response).to have_http_status(:unauthorized)
-      end
-
       it 'returns bad request with no event in the body' do
```

Comment:
This spec seems out of place. Maybe move to
- with a valid webhook key (existing context)
  - when event is missing (new context)